pandas/sparse/frame.py

```diff
@@ -302,7 +302,7 @@ def fillna(self, value=None, method=None, axis=0, inplace=False,
     # ----------------------------------------------------------------------
     # Support different internal representation of SparseDataFrame
 
-    def _sanitize_column(self, key, value):
+    def _sanitize_column(self, key, value, broadcast=False):
```

Comment:
I think it is OK to leave it in here, but please add a comment explaining this

How does an insert with duplicates currently work for SparseDataFrames ? Does it have the same bug?